ui/modal/info_modal_layouts.go

```diff
@@ -147,3 +150,26 @@ func allowUnspendUnmixedAcct(l *load.Load) []layout.Widget {
 		},
 	}
 }
+
+func ticketPriceErrorInfo(l *load.Load) []layout.Widget {
+	col := l.Theme.Color.GrayText2
+	return []layout.Widget{
+		func(gtx C) D {
+			text := l.Theme.Body1("Your wallet(s) needs to be synced before some of the staking functionality will available.")
```

Comment:
```suggestion
			text := l.Theme.Body1("Your wallet(s) need to be synced before some of the staking functionality will available.")
```

Reply:
Or better still, check if it's just one wallet available and use `wallet needs` or if multiple,  `wallets need`.